configs/config/pretrain/vision_transformer/swav/swav_deit_b_2nodes.yaml

```diff
@@ -42,14 +40,15 @@ config:
           std: [0.229, 0.224, 0.225]
       COLLATE_FUNCTION: cutmixup_collator
       COLLATE_FUNCTION_PARAMS: {
+        "ssl_method": "swav",
         "mixup_alpha": 1.0, # mixup alpha value, mixup is active if > 0.
         "cutmix_alpha": 1.0, # cutmix alpha value, cutmix is active if > 0.
         "prob": 1.0, # probability of applying mixup or cutmix per batch or element
         "switch_prob": 0.5, # probability of switching to cutmix instead of mixup when both are active
         "mode": "batch", # how to apply mixup/cutmix params (per 'batch', 'pair' (pair of elements), 'elem' (element)
         "correct_lam": True, # apply lambda correction when cutmix bbox clipped by image borders
         "label_smoothing": 0.1, # apply label smoothing to the mixed target tensor
-        "num_classes": 1000 # number of classes for target
+        "num_classes": 1 # number of classes for target
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.